### PR TITLE
Yukihiro matsumoto

### PR DESCRIPTION
**Pull request by mosmoo**
### Description

[Entry] General: Yukihiro Matsumoto #934
I added an image and wrote up content for Yukihiro Matsumoto's page on Codecademy. 

### Type of Change

- Adding a new entry

### Checklist

- [o] All writings are my own.
- [o] My entry follows the Codecademy Docs style guide.
- [o] My changes generate no new warnings.
- [o] I have performed a self-review of my own writing and code.
- [o] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [0] I have confirmed my changes are not being pushed from my forked `main` branch.
- [o] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [ ] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

**Comment by SSwiniarski** [comment]
@manasvislj, thanks for your contribution. Please remember to sign the CLA.

**Review by SSwiniarski — CHANGES_REQUESTED** [review]
@mosmoo. I performed a first review and left a couple of comments.

**Comment by Dusch4593** [comment]
> @manasvislj, thanks for your contribution. Please remember to sign the CLA.

@manasvislj link here: https://cla-assistant.io/Codecademy/docs?pullRequest=1158

**Comment by mosmoo** [comment]
> @mosmoo. I performed a first review and left a couple of comments.

Thank you! I have updated it, please let me know if I am good to proceed with the new changes.

**Review by SSwiniarski — APPROVED** [review]
@mosmoo, looks good for a second review.

File: content/general/concepts/Yukihiro_Matsumoto/Yukihiro_Matsumoto.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+  - 'Perl'
+CatalogContent:
+  - 'paths/code-foundations'
+---
+
+![Yukihiro Matsumoto (Matz)](https://raw.githubusercontent.com/Codecademy/docs/main/media/Yukihiro_Matsumoto.png
+
+**Yukihiro Matsumoto** (1965-present) a.k.a one of the open source evangelists of Japan, is a Japanese Software Programmer and Computer Scientist widely known for his work as the chief designer of Ruby. Matz blended parts of his favorite languages (Perl, Smalltalk, Eiffel, Ada, and Lisp) to form [Ruby](https://www.codecademy.com/resources/docs/ruby), a language of careful balance. Apart from Ruby, he is known for having released several open source products, including cmail, the emacs-based mail user agent, written entirely in emacs lisp.
+
+As of 2011, Matz is the Chief Architect of Ruby at [Heroku](https://en.wikipedia.org/wiki/Heroku "Heroku"), an online cloud platform-as-a-service in San Francisco. He was appointed to the role of technical advisor for VASILY Inc. starting in June 2014.
```
Comment (Dusch4593) [inline content/general/concepts/Yukihiro_Matsumoto/Yukihiro_Matsumoto.md:18]:
```suggestion
## Recent Activity

As of 2011, Matz is the Chief Architect of Ruby at [Heroku](https://www.heroku.com/), an online cloud platform-as-a-service. Starting in June 2014, he was appointed to the role of Technical Advisor for VASILY Inc.
```

I think both the roles mentioned should either be lowercase or uppercase.

File: content/general/concepts/Yukihiro_Matsumoto/Yukihiro_Matsumoto.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+
+## Writing
+
+In 2001, Matz published a book called _Ruby in a Nutshell_, and in 2008 he published _The Ruby Programming Language_. His demeanor has brought about a motto in the Ruby community: _"Matz is nice and so we are nice,"_ commonly abbreviated as MINASWAN.
```
Comment (Dusch4593) [inline content/general/concepts/Yukihiro_Matsumoto/Yukihiro_Matsumoto.md:22]:
```suggestion
In 2001, Matz published a book called "Ruby in a Nutshell." This was later expanded and revised into a new book called "The Ruby Programming Language," released in 2008 and co-authored by David Flanagan.
```

With the exception of the first mention of the entry's concept/term, let's refrain from using emphasis and, instead, wrap a published work in quotes.

Also, I think the last sentence fits better under the opening sentences.

**Review by Dusch4593 — CHANGES_REQUESTED** [review]
@mosmoo Finished with my review and left some comments/suggestions. 😄

**Comment by mosmoo** [comment]
Thank you so much for your edits and suggestions! I have also updated the branch with a resized image, please let me know I am good to proceed.

Thank you!

**Comment by Dusch4593** [comment]
Thanks! Before we go forward, I wanted to confirm that @mosmoo and @manasvislj are accounts owned/used by the same person?

**Comment by mosmoo** [comment]
> Thanks! Before we go forward, I wanted to confirm that @mosmoo and @manasvislj are accounts owned/used by the same person?

Yes both of them are my accounts. I am not sure why the edits were made using my old github, but I am working on fixing that!

**Review by Dusch4593 — APPROVED** [review]
@mosmoo LGTM! 👍🏻

I made a few edits in 61acffec0c1ac905af3433e23142b75e327e7b45 and 5d714b1382e4dc7f55af5efce71e0ad1e57c2561 including:

- Fixing spacing to `CatalogContent` in metadata at top.
- Changed/reformatted name of folder and file to get GitHub checks to pass.

**Comment by Dusch4593** [comment]
Congrats on making your first contribution to Codecademy Docs, @mosmoo! The entry can be viewed here: https://www.codecademy.com/resources/docs/general/yukihiro-matsumoto

Also, you are currently anonymous on the entry. If you'd like to receive credit, you'll need to add your GitHub username to your Codecademy profile, log out of your Codecademy profile, and then back in with your GitHub account:

Screen Shot 2022-10-28 at 2 07 12 PM

**Comment by yangc95** [comment]
@mosmoo This entry was moved to: https://www.codecademy.com/resources/docs/general/historical-technical-figures/yukihiro-matsumoto

Would you be able to make a small change to the introduction:

> **Yukihiro Matsumoto (Matz)** (1965-present) is a software programmer and computer scientist widely known for his work as the chief designer of the [Ruby](https://www.codecademy.com/resources/docs/ruby) programming language.

cc: @Dusch4593 @SSwiniarski

**Pull request merged**